pixart.js

```diff
@@ -1 +1,48 @@
-
+$(".brush").css("position", "<inline-block>");
+for (var i = 0; i < 3; i++) {
+  $("form").after($('<div />', {
+    "class": "brush",
+  }))};
+var allBrushes = $(".brush");
+allBrushes.css("display", "inline-block");
+var clickCounter = 0;
+$(document).ready(function() {
+  $("button").on("click", function(evt) {
+    var whichBrush = $(".brush")[clickCounter];
```

Comment:
When dealing with jQuery, remember to use `.eq` instead of `[]`, like so:

```js
var whichBrush = $(".brush").eq(clickCounter);
```

If you use brackets, you can't use any jQuery methods on the element in question. That is, this would work:

```js
$(".brush").eq(clickCounter).css()
```

...but not this:

```js
$(".brush")[clickCounter].css()
```